app/src/main/kotlin/org/fossify/calendar/helpers/CssColors.kt

```diff
@@ -0,0 +1,182 @@
+        "gray" to 0xFF808080.toInt(),
+        "green" to 0xFF008000.toInt(),
+        "greenyellow" to 0xFFADFF2F.toInt(),
+        "grey" to 0xFF808080.toInt(),
+        "honeydew" to 0xFFF0FFF0.toInt(),
+        "hotpink" to 0xFFFF69B4.toInt(),
+        "indianred" to 0xFFCD5C5C.toInt(),
+        "indigo" to 0xFF4B0082.toInt(),
+        "ivory" to 0xFFFFFFF0.toInt(),
+        "khaki" to 0xFFF0E68C.toInt(),
+        "lavender" to 0xFFE6E6FA.toInt(),
+        "lavenderblush" to 0xFFFFF0F5.toInt(),
+        "lawngreen" to 0xFF7CFC00.toInt(),
+        "lemonchiffon" to 0xFFFFFACD.toInt(),
+        "lightblue" to 0xFFADD8E6.toInt(),
+        "lightcoral" to 0xFFF08080.toInt(),
+        "lightcyan" to 0xFFE0FFFF.toInt(),
+        "lightgoldenrodyellow" to 0xFFFAFAD2.toInt(),
+        "lightgray" to 0xFFD3D3D3.toInt(),
+        "lightgreen" to 0xFF90EE90.toInt(),
+        "lightgrey" to 0xFFD3D3D3.toInt(),
+        "lightpink" to 0xFFFFB6C1.toInt(),
+        "lightsalmon" to 0xFFFFA07A.toInt(),
+        "lightseagreen" to 0xFF20B2AA.toInt(),
+        "lightskyblue" to 0xFF87CEFA.toInt(),
+        "lightslategray" to 0xFF778899.toInt(),
+        "lightslategrey" to 0xFF778899.toInt(),
+        "lightsteelblue" to 0xFFB0C4DE.toInt(),
+        "lightyellow" to 0xFFFFFFE0.toInt(),
+        "lime" to 0xFF00FF00.toInt(),
+        "limegreen" to 0xFF32CD32.toInt(),
+        "linen" to 0xFFFAF0E6.toInt(),
+        "magenta" to 0xFFFF00FF.toInt(),
+        "maroon" to 0xFF800000.toInt(),
+        "mediumaquamarine" to 0xFF66CDAA.toInt(),
+        "mediumblue" to 0xFF0000CD.toInt(),
+        "mediumorchid" to 0xFFBA55D3.toInt(),
+        "mediumpurple" to 0xFF9370DB.toInt(),
+        "mediumseagreen" to 0xFF3CB371.toInt(),
+        "mediumslateblue" to 0xFF7B68EE.toInt(),
+        "mediumspringgreen" to 0xFF00FA9A.toInt(),
+        "mediumturquoise" to 0xFF48D1CC.toInt(),
+        "mediumvioletred" to 0xFFC71585.toInt(),
+        "midnightblue" to 0xFF191970.toInt(),
+        "mintcream" to 0xFFF5FFFA.toInt(),
+        "mistyrose" to 0xFFFFE4E1.toInt(),
+        "moccasin" to 0xFFFFE4B5.toInt(),
+        "navajowhite" to 0xFFFFDEAD.toInt(),
+        "navy" to 0xFF000080.toInt(),
+        "oldlace" to 0xFFFDF5E6.toInt(),
+        "olive" to 0xFF808000.toInt(),
+        "olivedrab" to 0xFF6B8E23.toInt(),
+        "orange" to 0xFFFFA500.toInt(),
+        "orangered" to 0xFFFF4500.toInt(),
+        "orchid" to 0xFFDA70D6.toInt(),
+        "palegoldenrod" to 0xFFEEE8AA.toInt(),
+        "palegreen" to 0xFF98FB98.toInt(),
+        "paleturquoise" to 0xFFAFEEEE.toInt(),
+        "palevioletred" to 0xFFDB7093.toInt(),
+        "papayawhip" to 0xFFFFEFD5.toInt(),
+        "peachpuff" to 0xFFFFDAB9.toInt(),
+        "peru" to 0xFFCD853F.toInt(),
+        "pink" to 0xFFFFC0CB.toInt(),
+        "plum" to 0xFFDDA0DD.toInt(),
+        "powderblue" to 0xFFB0E0E6.toInt(),
+        "purple" to 0xFF800080.toInt(),
+        "rebeccapurple" to 0xFF663399.toInt(),
```

Comment:
RFC7986 explicitly mentions "CSS3" so `rebeccapurple` probably isn't supported because it's not part of the CSS3 named colors.

It was added later: https://www.w3.org/TR/css-color-4/#changes-from-3